### PR TITLE
#159951811 setup redux on Authors Haven front end

### PR DESCRIPTION
**Pull request by abulojoshua1**
### What does this PR do?
- Setup redux on Authors Haven frontend

### Description of Task to be completed?
- The **PR** sets up redux on the empty react project that was created.

### How should this be manually tested?
- Uncomment the code in [src/store/reducers/auth_reducer.js](https://github.com/andela/ah-frontend-taces/blob/ch-setup-redux-159951811/src/store/reducers/auth_reducer.js), [src/store/actions/actions.js](https://github.com/andela/ah-frontend-taces/tree/ch-setup-redux-159951811/src/store/actions) and  [src/App.js](https://github.com/andela/ah-frontend-taces/blob/ch-setup-redux-159951811/src/App.js). 
- Run the app using `npm start`
- Click the toggle button on the index page to view changes in react-redux states

### Any background context you want to provide?
- I installed redux
- Installed react-redux
- created  a store directory that holds reducers and actions
- created an index file in [store/reducers](https://github.com/andela/ah-frontend-taces/tree/ch-setup-redux-159951811/src/store/reducers) that combines all reducers

### What are the relevant pivotal tracker stories?
- [#159951811](https://www.pivotaltracker.com/story/show/159951811)

### CODE REVIEW

**Review by AGMETEOR — COMMENTED** [review]
Does this set up use REDUX DEV TOOLS?  Something like this 
 const store = createStore(
   reducer, /* preloadedState, */
+  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
 );

**Comment by abulojoshua1** [comment]
@AGMETEOR, thanks for identifying this. It will be quite helpful to us in this project. I will implement it ASAP and get back to you.

**Review by phillipseryazi — APPROVED** [review]

**Review by lytes20 — APPROVED** [review]

**Comment by geofrocker** [comment]
@abulojoshua1 Remove the commented code. Just add comments on how the files should be used

File: src/store/reducers/auth_reducer.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/*
```
Comment (geofrocker) [inline src/store/reducers/auth_reducer.js]: Remove this file

**Review by geofrocker — CHANGES_REQUESTED** [review]

File: src/store/reducers/auth_reducer.js (hunk 1 above)
Comment (abulojoshua1) [reply src/store/reducers/auth_reducer.js]: ok, I will do just that

**Review by ssewilliam — APPROVED** [review]

**Pull request merged**